app/controllers/buyers/service_contracts/bulk/base_controller.rb

```diff
@@ -24,6 +24,6 @@ def collection
   end
 
   def errors_template
-    'buyers/service_contracts/bulk/shared/errors.html'
+    'buyers/service_contracts/bulk/shared/errors'
```

Comment:
Example error that this change is fixing below.

I'm not quite sure why it was working before... 

```
Missing template buyers/service_contracts/bulk/shared/errors.html with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby, :slim, :coffee, :liquid]}.

Searched in:
  * "/opt/ci/workdir/app/views"
  * "/opt/ci/workdir/lib/developer_portal/app/views"
  * "/opt/ci/workdir/vendor/bundle/ruby/3.1.0/bundler/gems/swagger-ui_rails-f88bb8bed4fd/app/views"
  * "/opt/ci/workdir/vendor/bundle/ruby/3.1.0/bundler/gems/swagger-ui_rails-13968af8f284/app/views"

Did you mean?  buyers/service_contracts/bulk/shared/errors
               buyers/accounts/bulk/shared/errors
               buyers/applications/bulk/shared/errors
               developer_portal/layouts/error
               finance/provider/line_items/errors
               buyers/service_contracts/bulk/send_emails/create
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionview-7.0.8.6/lib/action_view/path_set.rb:49:in `find'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionview-7.0.8.6/lib/action_view/lookup_context.rb:127:in `find'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionview-7.0.8.6/lib/action_view/renderer/template_renderer.rb:49:in `determine_template'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionview-7.0.8.6/lib/action_view/renderer/template_renderer.rb:7:in `render'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionview-7.0.8.6/lib/action_view/renderer/renderer.rb:61:in `render_template_to_object'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionview-7.0.8.6/lib/action_view/renderer/renderer.rb:29:in `render_to_object'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionview-7.0.8.6/lib/action_view/rendering.rb:117:in `block in _render_template'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionview-7.0.8.6/lib/action_view/base.rb:270:in `in_rendering_context'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionview-7.0.8.6/lib/action_view/rendering.rb:116:in `_render_template'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionpack-7.0.8.6/lib/action_controller/metal/streaming.rb:216:in `_render_template'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionview-7.0.8.6/lib/action_view/rendering.rb:103:in `render_to_body'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionpack-7.0.8.6/lib/action_controller/metal/rendering.rb:158:in `render_to_body'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionpack-7.0.8.6/lib/action_controller/metal/renderers.rb:141:in `render_to_body'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionpack-7.0.8.6/lib/abstract_controller/rendering.rb:27:in `render'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionpack-7.0.8.6/lib/action_controller/metal/rendering.rb:139:in `render'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionpack-7.0.8.6/lib/action_controller/metal/instrumentation.rb:22:in `block (2 levels) in render'
/usr/share/ruby/benchmark.rb:311:in `realtime'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/activesupport-7.0.8.6/lib/active_support/core_ext/benchmark.rb:14:in `ms'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionpack-7.0.8.6/lib/action_controller/metal/instrumentation.rb:22:in `block in render'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionpack-7.0.8.6/lib/action_controller/metal/instrumentation.rb:91:in `cleanup_view_runtime'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/activerecord-7.0.8.6/lib/active_record/railties/controller_runtime.rb:34:in `cleanup_view_runtime'
/opt/ci/workdir/vendor/bundle/ruby/3.1.0/gems/actionpack-7.0.8.6/lib/action_controller/metal/instrumentation.rb:21:in `render'
/opt/ci/workdir/app/controllers/buyers/bulk_base_controller.rb:77:in `handle_errors'
```